docs/getting_started/build_an_app.md

```diff
@@ -0,0 +1,378 @@
+Furthermore, we can color based on the specie and change the kind of the plot to scatter.
+
+::::{tab-set}
+:::{tab-item} YAML
+:sync: yaml
+
+``` {code-block} yaml
+:emphasize-lines: 13-17
+sources:
+  penguin_source:
+    type: file
+    tables:
+      table_penguin: https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-07-28/penguins.csv
+
+targets:
+  - title: Penguins
+    source: penguin_source
+    views:
+      - type: hvplot
+        table: penguin_table
+        x: bill_length_mm
+        y: bill_depth_mm
+        kind: scatter
+        color: species
+```
+:::
+
+:::{tab-item} Preview
+:sync: preview
+
+![](../_static/getting_started/build_app_03.png)
+:::
+::::
+
+
+Let's now set up two filters based on sex and the island of the penguins, which is done by adding a pipeline with these filters.
+
+::::{tab-set}
+:::{tab-item} YAML
+:sync: yaml
+
+``` {code-block} yaml
+:emphasize-lines: 7-15
+
+sources:
+  penguin_source:
+    type: file
+    tables:
+      penguin_table: https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-07-28/penguins.csv
+
+pipelines:
+  penguin_pipeline:
+    source: penguin_source
+    table: penguin_table
+    filters:
+      - type: widget
+        field: sex
+      - type: widget
+        field: island
+
+targets:
+  - title: Penguins
+    pipeline: penguin_pipeline
+    views:
+      - type: hvplot
+        x: bill_length_mm
+        y: bill_depth_mm
+        kind: scatter
+        color: species
+```
+:::
+
+:::{tab-item} Preview
+:sync: preview
+
+![](../_static/getting_started/build_app_04.png)
+:::
+::::
+
+We can even expand the views with more plots like histograms of some of the penguin's features.
+
+::::{tab-set}
+:::{tab-item} YAML
+:sync: yaml
+
+``` {code-block} yaml
+:emphasize-lines: 26-34
+
+sources:
+  penguin_source:
+    type: file
+    tables:
+      penguin_table: https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-07-28/penguins.csv
+
+pipelines:
+  penguin_pipeline:
+    source: penguin_source
+    table: penguin_table
+    filters:
+      - type: widget
+        field: sex
+      - type: widget
+        field: island
+
+targets:
+  - title: Penguins
+    pipeline: penguin_pipeline
+    views:
+      - type: hvplot
+        x: bill_length_mm
+        y: bill_depth_mm
+        kind: scatter
+        color: species
+      - type: hvplot
+        kind: hist
+        y: bill_length_mm
+      - type: hvplot
+        kind: hist
+        y: bill_depth_mm
+      - type: hvplot
+        kind: hist
+        y: body_mass_g
+
+```
+:::
+
+:::{tab-item} Preview
+:sync: preview
+
+![](../_static/getting_started/build_app_05.png)
+:::
+::::
+
+
+The layout of the views is in a column where we can't even see the last plot, and the dashboard is non-responsive.
```

Comment:
```suggestion
The default layout we get is less than ideal for this case since it cuts off one of our plots, leaves a lot of empty space and does not resize responsively.
```